test/resources/case13_templatization/case13_fromsecret.yaml

```diff
@@ -3,6 +3,7 @@ kind: ConfigurationPolicy
 metadata:
   name: tmplt-policy-secret-duplicate
 spec:
+  pruneObjectBehavior: DeleteAll
```

Comment:
Is this change needed?

Reply:
yes

Reply:
I had requested this :)

If this test doesn't have `pruneObjectBehavior` set, then the template error definitely shouldn't cause anything to be removed (because now, things are only removed when that is set). The test is meant to answer the question "Will objects be removed when there is a templating error, and I generally want objects to be pruned?" with "No, objects will not be removed".